palantirnet/st-bernards
# MSSB-87: Update style for active menu element

## JIRA Ticket(s)

- See ticket: https://palantir.atlassian.net/browse/MSSB-87

## Description

Update active menu state

## Testing instructions

1. Read ticket
1. Run `styleguide`
1. Run `drush cr && drush cim -y && drush cr`
1. Visit URL's in ticket and verify.
